pages/glossary/visual-context.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Visual context
+key: visual-context
+---
+
+The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
+
+It may include, but is not limited to, headings, text in the same sentence or paragraph.
```

Comment:
This is open ended. That makes this ambiguous, which isn't allowed in applicability and expectations.

Reply:
That's not necessarily true. Things can be unambiguous and open ended. Even the phrase "describes the purpose of" is open ended, but I think we can all agree that it's unambiguous enough for the purpose of an ACT rule. It's very subjective though, which is allowed in expectations.